core/geotime/src/test/java/org/locationtech/geowave/core/geotime/store/query/aggregate/AbstractVectorAggregationTest.java

```diff
@@ -19,7 +18,9 @@
 import org.locationtech.jts.geom.Coordinate;
 import org.locationtech.jts.geom.Geometry;
 import org.opengis.feature.simple.SimpleFeature;
-import com.google.common.collect.Lists;
+import org.opengis.feature.simple.SimpleFeatureType;
+import java.util.Date;
+import java.util.List;
```

Comment:
need to go thru the import reorganizations and change them back, they were fallout from a different formatter.